cfn-resources/autogen/schemas/mongodb-atlas-x509authenticationdatabaseuser.json

```diff
@@ -1,292 +0,0 @@
-{
```

Comment:
Should we also be deleting x509 / other resource schemas as well?

Reply:
I don't know which other schemas you are referring to. Could you share a link to these schemas, please? I will look to see if we should remove those as well. Thanks!

Reply:
I will open a follow-up PR to address this comment if needed.